### DISCUSSION TITLE
deploy to hubs aws error return ROLLBACK_COMPLETE

### DISCUSSION THREAD

**Discussion by jsdude0220** [mozilla/hubs#3834] [Hubs Cloud Questions] [answered]
The maximum number of internet gateways has been reached. 
The maximum number of VPCs has been reached.

I have got those errors when create the new stack and deploy to my old hubs aws domain.

**Comment by pattersonbl2** [comment]
Hey, are you still having issues with this.  it seems like you are deploying a new stack with left over resources.  When deploying a stack aws normally doesn't delete resources.

**Comment by Dayk0** [comment, marked as answer]
I have the same problem, if someone managed to find the solution ...: https://prnt.sc/1md4wqt

**Reply by pattersonbl2** [reply]
This error means that the quota for Igw and VPC has been reached in your account. You would have to reach out to AWS support to increase the quota or delete some unused resource.  https://docs.aws.amazon.com/vpc/latest/userguide/amazon-vpc-limits.html
